source/api/presentation/3.0/index.md

```diff
@@ -826,40 +860,40 @@ Additional motivations may be added to the Annotation to further clarify the int
 
 This section describes features applicable to all of the Presentation API content. For the most part, these are features of the JSON-LD specification that have particular uses within the API and recommendations about URIs to use.
 
-### 4.1. HTTPS URI Scheme
-
-It is strongly _RECOMMENDED_ that all URIs use the HTTPS scheme, and be available via that protocol. All URIs _MUST_ be either HTTPS or HTTP, abbreviated to "HTTP(S)" in this specification.
+### 4.1. Resource Representations
 
-### 4.2. URI Representation
-
-Resource descriptions _SHOULD_ be embedded within higher-level descriptions, and _MAY_ also be available via separate requests from HTTP(S) URIs linked in the responses. These URIs are in the `id` property for the resource. Links to resources _MUST_ be given as a JSON object with the `id` property and at least one other property, typically either `type`, `format` or `profile` to give a hint as to what sort of resource is being referred to. Other URI schemes _MAY_ be used if the resource is not able to be retrieved via HTTP(S).
+Resource descriptions _SHOULD_ be embedded within the JSON description of parent resources, and _MAY_ also be available via separate requests from the HTTP(S) URI given in the resource's `id` property. Links to resources _MUST_ be given as a JSON object with the `id` property and at least one other property, typically either `type`, `format` or `profile` to give a hint as to what sort of resource is being referred to.
 
 ``` json-doc
 {
-  "thumbnail": [
-    { "id": "https://example.org/images/thumb1.jpg", "type": "Image" }
+  "rendering": [
+    { 
+      "id": "https://example.org/content/book.pdf", 
+      "type": "Text",
+      "label": "Example Book (pdf)",
+      "format": "application/pdf"
+    }   
   ]
 }
 ```
 
-### 4.3. Repeatable Properties
+### 4.2. Properties with Multiple Values
 
 Any of the properties in the API that can have multiple values _MUST_ always be given as an array of values, even if there is only a single item in that array.
 
 ``` json-doc
 {
-  "seeAlso": [
-    { "id": "https://example.org/images/thumb1.jpg", "type": "Image" },
-    { "id": "https://example.org/videos/thumb1.pmg", "type": "Video" }   
+  "thumbnail": [
+    { "id": "https://example.org/images/thumb1.jpg", "type": "Image" }
   ]
 }
 ```
 
-### 4.4. Language of Property Values
+### 4.3. Language of Property Values
 
-Language _MAY_ be associated with strings that are intended to be displayed to the user for the `label` and `summary` fields, plus the `label` and `value` fields of the `metadata` and `requiredStatement` objects.
+Language _MAY_ be associated with strings that are intended to be displayed to the user for the `label` and `summary` properties, plus the `label` and `value` properties of the `metadata` and `requiredStatement` objects.
```

Comment:
As we now have a spec where it's hard not to declare a language for a string, should this _MAY_ be a _SHOULD_?

Reply:
If you didn't know the language (because of some source data where language is not specified, for example) then you probably shouldn't try to associate a language. I could see "_SHOULD_ if known" but I wonder whether _SHOULD_ on its own is too strong?

Reply:
I'm happy with _SHOULD_ -- it is the International IIF after all. 
I'm not happy with an unenforcable conditional.  That's just MAY, as you wouldn't want to throw a warning if it was missing, because you can never tell if the "if known" part is in effect or not.  Propose a new issue.

Reply:
--> https://github.com/IIIF/api/issues/1492